src/node/mini_miner.h

```diff
@@ -162,10 +169,9 @@ class MiniMiner
     std::optional<CAmount> CalculateTotalBumpFees(const CFeeRate& target_feerate);
 
     /** Construct a new block template with all of the transactions and calculate the order in which
-     * they are selected. Returns the sequence number (lower = selected earlier) with which each
-     * transaction was selected, indexed by txid, or an empty map if it cannot be calculated.
+     * they are selected and the packages fee rate and sizes.
```

Comment:
The sentence used to end at `they are selected.`  Now it's a very long run on sentence.  It would be better to break it into two sentences or shorten it.

Reply:
fixed.